bciers/apps/reporting/src/app/components/operations/OperationReview.tsx

```diff
@@ -1,144 +1,197 @@
 "use client";
-import {
-  Box,
-  Typography,
-  Button,
-  Paper,
-  Grid,
-  List,
-  ListItem,
-  ListItemText,
-  ListItemIcon,
-} from "@mui/material";
-import InfoIcon from "@mui/icons-material/Info";
-import SyncIcon from "@mui/icons-material/Sync";
-import CheckCircleOutline from "@mui/icons-material/CheckCircleOutline";
-import MultiStepHeader from "@bciers/components/form/components/MultiStepHeader";
-import { actionHandler } from "@bciers/actions";
-import FormBase from "@bciers/components/form/FormBase";
+
+import React, { useEffect, useState } from "react";
+import MultiStepFormWithTaskList from "@bciers/components/form/MultiStepFormWithTaskList";
+import { RJSFSchema } from "@rjsf/utils";
+import { useRouter } from "next/navigation";
 import {
   operationReviewSchema,
   operationReviewUiSchema,
 } from "@reporting/src/data/jsonSchema/operations";
+import { TaskListElement } from "@bciers/components/navigation/reportingTaskList/types";
 import safeJsonParse from "@bciers/utils/safeJsonParse";
+import { actionHandler } from "@bciers/actions";
+import dayjs from "dayjs";
 
 interface Props {
   formData: any;
   version_id: number;
+  reportingYear: {
+    reporting_year: number;
+    report_due_date: string;
+    reporting_window_end: string;
+  };
+  allActivities: { id: number; name: string }[];
+  allRegulatedProducts: { id: number; name: string }[];
```

Comment:
These could be just a `string[]` with the activities/products' names, since that's how the backend keys things (for now)

Reply:
I added Ids, as that was required by MultiSelect widget for autocomplete when auto populating existing data